packages/paima-sdk/paima-mw-core/src/helpers/posting.ts

```diff
@@ -65,6 +65,39 @@ export async function updateFee(): Promise<void> {
   }
 }
 
+//
+// reCaptcha to validate batcher submissions
+//
+// To enable validation:
```

Comment:
the fact this comment block is duplicated in this function makes me think even more this should be in the docs instead